pkg/cmd/roachtest/test_impl.go

```diff
@@ -494,17 +494,17 @@ func (t *testImpl) failureMsg() string {
 	return b.String()
 }
 
-// failuresContainError returns true if any of the errors in any of
-// the given failures matches the reference error.
-func failuresContainsError(failures []failure, refError error) bool {
+// failuresMatchingError returns an error of the same type as
```

Comment:
Nit: "returns an error" is a bit subtle since `errors.As` updates `refError` via a side-effect. For the record, the official description of `errors.As` makes it more precise,

> As finds the first error in err's tree that matches target, and if one is found, sets target to that error value and returns true. Otherwise, it returns false.

Reply:
Good point, updated to match `errors.As`'s documentation.